### PR TITLE
Update query intervals

### PR DESCRIPTION
**Pull request by mayerm94**
Update timing intervals to match what is running on ec2
Pull requests into this repository require the following checks to be completed by the submitter and reviewers.

**Submitter:**
- [x] Administrative tasks are complete.
  - This pull request describes why these changes were made.
  - The JIRA ticket links to this PR.
  - The JIRA ticket for this PR is: https://oncprojectracking.healthit.gov/support/browse/LANTERN-104
  - vendor/ directories have been updated
    - New dependencies have been added
    - Local dependencies have been updated
- [x] Tests are complete. NA
  - Tests are included and test edge cases.
  - Tests have been run locally and pass.
- [x] Code review has been performed.
  - Code diff has been done and checked to ensure only expected code is being committed.
  - A linter for the language has been run.

**Primary Reviewer:** @edeyoung 

- [x] Code works.
  - Code has been run locally and the basic functionality affected by the PR has been checked.
  - Edge cases have been checked to ensure behavior is as expected.
- [x] Code review has been performed.
  - Code is maintainable, reusable, efficient, and correct.
  - Code accomplishes the tasks purpose.
  - Code follows style guidance appropriate for the language, including passing linter checks.
  - Code is well commented (which does not mean verbosely commented).
  - vendor/ directories have been updated
    - New dependencies have been added
    - Local dependencies have been updated
- [x] Tests are complete.
  - Tests are included and test edge cases.
  - Tests have been run locally and pass.

**Secondary Reviewer:**

- [x] Code works.
  - Code has been run locally and the basic functionality affected by the PR has been checked.
  - Edge cases have been checked to ensure behavior is as expected.
- [x] Code review has been performed.
  - Code accomplishes the tasks purpose.
- [x] Tests are complete.
  - Tests have been run locally and pass.

**Merger:**

Prior to merging, ensure that all checks have been completed. Delete the branch after the merge is complete.

### CODE REVIEW

File: endpoints/README.md [hunk 1]
```diff
@@ -13,6 +13,10 @@ The FHIR Endpoint Querier reads the following environment variables:
 
   Default value: endpointQuerierLog.json
 
+* **LANTERN_ENDPTQRY_QUERY_INTERVAL**: Number of minutes to wait between queries (Note: actual time between queries will be greater since the time it takes to run the queries is non-zero)
```
Comment (edeyoung) [inline endpoints/README.md]: state that this is in minutes.

File: endpoints/README.md [hunk 2]
```diff
@@ -13,6 +13,10 @@ The FHIR Endpoint Querier reads the following environment variables:
...
+
+  Default value: 10m
```
Comment (edeyoung) [inline endpoints/README.md]: change this to `10` instead of `10m`

File: prometheus.yml [hunk 3]
```diff
@@ -1,7 +1,7 @@
 # my global config
 global:
-  scrape_interval:     15s # Set the scrape interval to every 15 seconds. Default is every 1 minute.
-  evaluation_interval: 15s # Evaluate rules every 15 seconds. The default is every 1 minute.
+  scrape_interval:     240s # Set the scrape interval to every 15 seconds. Default is every 1 minute.
```
Comment (edeyoung) [inline prometheus.yml]: recommend setting this to minutes to make it more readable: `4m`

Comment (edeyoung) [reply prometheus.yml]: also change the comment text

File: endpoints/README.md (hunk 1 above)
Comment (mayerm94) [reply endpoints/README.md]: "Number of minutes to wait...." doesn't satisfy that?

Comment (edeyoung) [reply endpoints/README.md]: it definitely does... must have read over that somehow.
